<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_noise</class>
 <widget class="QWidget" name="Form_noise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_s">
       <property name="text">
        <string>椒盐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>噪声程度（0-1）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_sn"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_g">
       <property name="text">
        <string>高斯</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>均值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_mean"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>标准差</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_sigma"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pb_r">
       <property name="text">
        <string>随机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>噪点数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_rn"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
